docs/config.md

```diff
@@ -257,3 +257,9 @@ Instead of tagging, you also have the option to add metadata to instances. This
         securityGroups:
    ...
    ```
+
+## Timeout settings
+During some heavy workload cloud, the time for create and delete openstack instance might takes long time, by default it's 5 minute.
```

Comment:
Since those parameters are used in controller, I think the doc here should also be updated by clarifying how to update the deployment yaml file by adding those variables?

Reply:
Do we really need those? it's general stuff of k8s?